tests/e2e/vsan_test.go

```diff
@@ -57,6 +57,10 @@ func (s *VsanTestSuite) SetUpSuite(c *C) {
 	}
 }
 
+func (s *VsanTestSuite) SetUpTest(c *C) {
+	s.policyList = []string{}
```

Comment:
I put it in the setup is to make sure each test is running from a clean state. I think either way works. So I will keep it as it is.